packages/vite/src/node/__tests__/plugins/importGlob/fixture-a/index.ts

```diff
@@ -67,3 +67,5 @@ export const cleverCwd2 = import.meta.glob([
   '../fixture-b/*.ts',
   '!**/index.ts',
 ])
+
+export const customBase = import.meta.glob('**/*.ts', { base: './' })
```

Comment:
```suggestion
export const customBase = import.meta.glob('**/*.ts', { base: './' })

export const customRootBase = import.meta.glob('**/*.ts', { base: '/fixture-b' })
```
Let's test this one as well.

Reply:
Thanks. I have added a test.
[5db0640](https://github.com/vitejs/vite/pull/18510/commits/5db0640b20913245a861dad03d2e31529d127bf2)